jobs/build.Jenkinsfile

```diff
@@ -274,6 +274,24 @@ lock(resource: "build-${params.STREAM}") {
         if (prevBuildID == buildID) {
             currentBuild.result = 'SUCCESS'
             currentBuild.description = "${build_description} 💤 (no new build)"
+
+            def builds = readJSON file: "builds/builds.json"
```

Comment:
should we do a sanity check here to confirm 100% that builds.builds[0] matches the build ID we expect it to?